assets/quick-order-list.css

```diff
@@ -13,6 +13,8 @@ quick-order-list .quantity__button {
 
 .quick-order-list__container {
   padding-bottom: 4rem;
+  max-height: 90rem;
+  overflow-y: auto;
```

Comment:
We need to rethink about it because it causes an issue with scrolling 👇

https://screenshot.click/25-36-u81os-rw42j.mp4

Reply:
ah, I recall this actually. Let me drop a short video here why I was initially against adding max-height and overflow-y. 
https://screenshot.click/25-02-hwmw2-3h4x2.mp4

Of course it's up to @edmund-teh. I think if we decide to go with max-height and scrolling bar then we need to add paddings on the right for both the table and total bar and somehow make it work equally for users who use touchpad and the mouse.

Reply:
Were we okay that scrolling bar goes over the price 🤔 ?
<img width="1298" alt="Screenshot 2023-09-25 at 12 33 25 PM (2)" src="https://github.com/Shopify/dawn/assets/105315663/2512e787-32ea-44c4-960b-1c2611bfa9e2">

Reply:
I also I think if we add a max height, we need to add a scroll bar (and ideally we do that to avoid a super long section)

 I also believe a scroll bar is an accessibility requirement:

https://www.w3.org/TR/WCAG20-TECHS/SCR36.html
https://screenshot.click/25-16-p115q-9nbzd.png

Reply:
@eugenekasimov dont be sorry :) it's important to discuss all the pros and cons

Reply:
I really like an approach we use for the cart-drawer when we see a scrolling bar all the time no matter what settings/devices are 👍